documentation/config_file_puppetserver.markdown

```diff
@@ -73,6 +73,19 @@ The `puppetserver.conf` file contains settings for Puppet Server software. For a
 * The `profiler` settings configure profiling:
 
     * `enabled`: If this is set to `true`, Puppet Server enables profiling for the Puppet Ruby code. The default is `false`.
+    
+* The `product` settings configure update checking and analytics data collection:
```

Comment:
Would it be better to document that the `product` settings go into a `product.conf` file rather than listing it under the `puppetserver` conf section.  Doesn't seem like a puppetserver-specific thing anyway, like `webserver` and others.  Seems like for users that want to opt-out of metrics collection but are otherwise good with using the defaults for everything else that goes into the `puppetserver.conf` file that they'd be better protected against a breaking change on upgrade if they put the `product.check-for-updates` setting in a new file that won't conflict with the file in the package.  Note that in today's Puppet Server, the `product` section doesn't exist in any conf files by default.